CMakeLists.txt

```diff
@@ -66,7 +66,7 @@ target_precompile_headers(YimMenu PRIVATE "${SRC_DIR}/common.hpp")
 target_link_libraries(YimMenu PRIVATE pugixml minhook AsyncLogger dbghelp imgui cpr lua_static asmjit::asmjit)
 
 # Warnings as errors
-set_property(TARGET YimMenu PROPERTY COMPILE_WARNING_AS_ERROR ON)
+set_property(TARGET YimMenu PROPERTY COMPILE_WARNING_AS_ERROR OFF)
```

Comment:
lol

Reply:
I'm not that experienced, so I chose the simplest way so that the error message no longer appears when compiling. Now I know that I just have to remove the comments.

Reply:
I'm not that experienced, so I chose the simplest way so that the error message no longer appears when compiling. Now I know that I just have to remove the comments.

Reply:
this work witrhout local online save file??